GATEWAY/src/providers/tilesApi.service.ts

```diff
@@ -12,12 +12,25 @@ export class TilesApi {
   private virtualTiles: VirtualTile[] = [];
   private loginData: LoginData;
   private activeApp: Application;
+  flagThen: boolean = false;
+  flagCatch: boolean = false;
 
   constructor(private http: Http,
               private storage: Storage) {
   }
 
   /**
+   * The following code will mailny be used for getting private parameters
```

Comment:
You mean ble?
If so, it is for the same reason
I need to be able to get these variables to properly test the class